src/liballoc/collections/btree/set.rs

```diff
@@ -109,6 +109,83 @@ pub struct Range<'a, T: 'a> {
     iter: btree_map::Range<'a, T, ()>,
 }
 
+/// Core of SymmetricDifference and Union.
+/// More efficient than btree.map.MergeIter,
+/// and crucially for SymmetricDifference, nexts() reports on both sides.
+#[derive(Clone)]
+struct MergeIterInner<I>
+where
+    I: Iterator,
+    I::Item: Copy,
```

Comment:
The `where` style in the new code does not match the existing code in the file.

What do we normally do? rustfmt? I'd prefer to keep the existing style, we keep the same file consistent.

Reply:
I normally rustfmt everything, and there are only 2 places where set.rs doesn't follow suit (but that's because a lot of code is commented out in my working copy).

Note sure what the existing style is though. They're all functions instead of types, and while most have one line per clause, BTreeSet::range has all clauses on one line, split_off squashes further. map.rs shows them on structs, but is in doubt about the number of clauses per line.

So it seems we indent where always once (or twice, but not trice for nested functions) and put the first clause behind.